server/data/incidentClient.test.js

```diff
@@ -119,11 +125,14 @@ test('submitReport', () => {
   incidentClient.submitReport('user1', 'booking1', 'date1')
 
   expect(db.query).toBeCalledWith({
-    text: `update report r set status = $1, submitted_date = $2
-    where user_id = $3
-    and booking_id = $4
-    and status = $5
-    and r.sequence_no = (select max(r2.sequence_no) from report r2 where r2.booking_id = r.booking_id and user_id = r.user_id)`,
+    text: `update report r
+            set status = $1
+            ,   submitted_date = $2
+            ,   updated_date = now()
+          where user_id = $3
+          and booking_id = $4
```

Comment:
v minor - consistency with the table alias?

Reply:
Thanks, I'll make consistent in a future PR.